### PR TITLE
Logging

### PR DESCRIPTION
**Pull request by TimCsaky**
# Description

addresses Ticket [1419](https://apps.nrs.gov.bc.ca/int/jira/browse/SHOWCASE-1419?)

We haven't been logging the template file type for render requests that use a cached template.

This PR adds this data to the logs (as well as the hash of the cached template) for new logs.

 
## Types of changes

Bug fix (non-breaking change which fixes an issue)

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have checked that unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have updated the OpenAPI 3.0 `v*.api-spec.yaml` documentation (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by bcgov-nr-csst** [comment]
SonarQube Analysis: https://sonarqube-idcqvl-tools.pathfinder.gov.bc.ca/dashboard?id=common-document-generation-service-pr-38

**Comment by bcgov-nr-csst** [comment]
SonarQube Analysis: https://sonarqube-idcqvl-tools.pathfinder.gov.bc.ca/dashboard?id=common-document-generation-service-pr-38

File: app/src/middleware/carboneCopy.js [hunk 1]
```diff
@@ -153,7 +155,35 @@ const initializeApiTracker = (app, basePath) => {
...
+  });
+
+  morgan.token('existingTemplate', req => {
+    try {
+      // if using existing template return the hash else return '-'
+      if (req._op == 'GENERATE_FROM_TEMPLATE') {
+        // get file hash
+        const split = req.url.split('/');
+        const hash = split[2];
+        return hash;
```
Comment (jujaga) [inline app/src/middleware/carboneCopy.js:182]:
```js
return split[2];
```

**Comment by bcgov-nr-csst** [comment]
SonarQube Analysis: https://sonarqube-idcqvl-tools.pathfinder.gov.bc.ca/dashboard?id=common-document-generation-service-pr-38

**Pull request merged**